### PR TITLE
breadcrumbs review

### PR DESCRIPTION
**Pull request by pascalalferink**
Hi everybody,
Today I wrote my first repository for CastleCSS: **castlecss-breadcrumbs**. I am looking forward for your feedback, 

To be more specific, I am wandering what you all think of:

1: the variable of the separator is named: **$castlecss-breadcrumbs-separator** Do you think all variables should be prefixed with castlecss or not?
2: In breadcrumbs.scss I made a check if the variable exists, do you think a check like this should be in the top of the document, of is the current place correct?
3:  If a user wants to place a specific separator, he can add the attribute **data-castlecss-breadcrumbs-separator** to the list-item. The value of this attribute is used in the css to place the separator. Is this a good way?

To view a working example of the breadcrumbs, go to breadcrumb feature-branch in CastleCSS-Docs
https://github.com/CastleCSS/castlecss-docs/tree/feature/castlecss-breadcrumbs

Kind regards, 
Pascal Alferink

### CODE REVIEW

File: sass/breadcrumbs.scss [hunk 1]
```diff
@@ -1,25 +1,32 @@
...
-			}
+.breadcrumb {
+	display: inline;
+	white-space: nowrap;
+
+	&:after {
+		display: inline;
+		
+		@if variable-exists(castlecss-breadcrumbs-separator) {
+			content: $castlecss-breadcrumbs-separator; // default separator character between crumbs.
```
Comment (amberjasmijn) [inline sass/breadcrumbs.scss]: I'm in doubt about adding 'castlecss' to the variables. It means: more writing throughout the project, but makes clear it is a castlecss variable, not a user variable. My preference is to leave it behind.

File: sass/breadcrumbs.scss [hunk 2]
```diff
@@ -1,25 +1,32 @@
-.castlecss-breadcrumbs{
-	
-	.breadcrumb-wrap {
-		display: block;		
+.castlecss-breadcrumbs {
```
Comment (amberjasmijn) [inline sass/breadcrumbs.scss]: @pascalalferink why did you prefixed breadcrumbs with 'castlecss'?

File: sass/breadcrumbs.scss [hunk 3]
```diff
@@ -1,25 +1,32 @@
...
-				}
-			}
+.breadcrumb {
+	display: inline;
+	white-space: nowrap;
+
+	&:after {
+		display: inline;
+		
+		@if variable-exists(castlecss-breadcrumbs-separator) {
```
Comment (amberjasmijn) [inline sass/breadcrumbs.scss]: Seems like a legit place to check if there's a variable!

File: sass/breadcrumbs.scss (hunk 1 above)
Comment (dariusrosendahl) [reply sass/breadcrumbs.scss]: Agreed, adding castlecss- to the variables seems a bit overkill to me. We're not doing it for other variables either, lets leave it out?

File: sass/breadcrumbs.scss (hunk 2 above)
Comment (pascalalferink) [reply sass/breadcrumbs.scss]: @amberjasmijn Thanks for your reply, the prefix will be removed in the next version.

File: sass/breadcrumbs.scss (hunk 3 above)
Comment (pascalalferink) [reply sass/breadcrumbs.scss]: @amberjasmijn oke, I leave it here

File: sass/breadcrumbs.scss (hunk 1 above)
Comment (pascalalferink) [reply sass/breadcrumbs.scss]: @amberjasmijn , @dariusrosendahl Thanks for your replies, the prefix will be removed in the next version.

**Pull request merged**